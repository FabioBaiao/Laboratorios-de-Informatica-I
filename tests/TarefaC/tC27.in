Aca
aca
aca
aca
aca
aaa
2 5 S
AAAAADAADAAAAAL