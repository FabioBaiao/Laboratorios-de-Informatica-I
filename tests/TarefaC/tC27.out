0 5
FIM 15
